src/nbl/video/CVulkanPhysicalDevice.cpp

```diff
@@ -403,14 +403,19 @@ std::unique_ptr<CVulkanPhysicalDevice> CVulkanPhysicalDevice::create(core::smart
 
         if (!vulkan12Properties.shaderSignedZeroInfNanPreserveFloat32)
             return nullptr;
-        if (!vulkan12Properties.shaderDenormPreserveFloat32)
-            return nullptr;
-        if (!vulkan12Properties.shaderDenormFlushToZeroFloat32)
-            return nullptr;
-        if (!vulkan12Properties.shaderRoundingModeRTEFloat32)
-            return nullptr;
-        if (!vulkan12Properties.shaderRoundingModeRTZFloat32)
-            return nullptr;
+        properties.limits.shaderSignedZeroInfNanPreserveFloat64 = vulkan12Properties.shaderSignedZeroInfNanPreserveFloat64;
```

Comment:
remove this single line

Reply:
what!?

Reply:
oh makes sense. in that case `shaderSignedZeroInfNanPreserveFloat64` should be added as well